Solutions/Corvus.Identity.Certificates/Corvus.Identity.Certificates.csproj

```diff
@@ -19,15 +16,11 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="Endjin.RecommendedPractices.GitHub" Version="2.1.9">
-	    <PrivateAssets>all</PrivateAssets>
-	    <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-	  </PackageReference>
-	  <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
-	  <PackageReference Include="PolySharp" Version="1.14.1">
+	  <PackageReference Include="Endjin.RecommendedPractices.GitHub" Version="2.1.18">
 	    <PrivateAssets>all</PrivateAssets>
 	    <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
 	  </PackageReference>
+	  <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="[8.0.*,)" />
```

Comment:
These `[,)` style version specs turned out to be a mistake. I misunderstood what they did back when I introduced them, and I've been gradually removing them since.

At the time I thought these got written into the NuGet package. Turns out they don't: they get resolved at compile time to a specific version, meaning that the build is no longer deterministic. (If a new version comes out, you'll get that, meaning that the resulting NuGet package is different from what would have been built before that new version came out despite no change in source code.)

This form of package spec really only makes sense in applications (and test suites); in a library, it turns out to be no more useful than hard-coding it, with the added feature of making builds non-repeatable.